openshift_pipelines/working-with-pipelines-using-the-developer-perspective.adoc

```diff
@@ -0,0 +1,22 @@
+[id='op-working-with-pipelines-using-developer-perspective']
+= Working with {op-title} using the Developer
+include::modules/openshift-pipelines-attributes.adoc[]
+include::modules/common-attributes.adoc[]
+:context: working-with-pipelines-using-developer-perspective
+toc::[]
+
+You can use the *Developer* perspective of the OpenShift web console to create CI/CD pipelines for your software delivery process while creating an application on OpenShift Container Platform.
+
+After you create pipelines, you can view and visually interact with your deployed pipelines.
+
+NOTE: The .NET Core runtime is not yet supported by the pipeline templates.
+
+[Discrete]
+=== Prerequisites
+
+* You have access to an OpenShift Cluster and have logged in to the link:https://docs.openshift.com/container-platform/latest/web_console/web-console.html#web-console[web console].
+* You have cluster administrator privileges to install operators and have link:TBD[installed the OpenShift Pipelines Operator].
```

Comment:
Just a reminder to update the link

Reply:
I have commented out this content for now. I will add it back when we have the Installing Pipeline procedure merged.